src/idl_gen_rust.cpp

```diff
@@ -327,8 +327,9 @@ bool GenerateRustModuleRootFile(const Parser &parser,
   }
   CodeWriter code("  ");
   // TODO(caspern): Move generated warning out of BaseGenerator.
+  
   code +=
-      "// Automatically generated by the Flatbuffers compiler. "
+      "// @generated \n// Automatically generated by the Flatbuffers compiler. "
```

Comment:
nit: This should go below the human readable line

Reply:
also, this code doesn't use `\n`, use another `code +=`

Reply:
Sure, will do that.
Could you confirm if idl_gen_rust.cpp is the only file to modify  for modifying all rust generated files.

Reply:
You should replace these lines with a call to the `FlatBuffersGeneratedWarning()`